src/index.js

```diff
@@ -14,4 +13,3 @@ root.render(
 // If you want to start measuring performance in your app, pass a function
```

Comment:
- [ ] Please remove this comment that was originally in the boilerplate. Since you deleted reportWebVitals, this comment has no significance being here :+1: